client/ipa-client-automount.in

```diff
@@ -319,17 +319,37 @@ def uninstall(fstore, statestore):
         running = statestore.restore_state('rpcidmapd', 'running')
         rpcidmapd = services.knownservices.rpcidmapd
         if not enabled:
-            rpcidmapd.disable()
+            try:
+                rpcidmapd.disable()
+            except Exception as e:
+                logger.error(
+                    "%s failed to disable automatic startup of the %s daemon",
+                    rpcidmapd.service_name, e)
```

Comment:
Having second look... Is this correct order of arguments @fcami ?

Reply:
Not really, I'll fix it, good catch!